fetch.bs

```diff
@@ -346,6 +346,9 @@ following <a for=struct>items</a>: [[RESOURCE-TIMING]] [[NAVIGATION-TIMING]]
  <dt><dfn export for="response body info" id="fetch-timing-info-decoded-body-size">decoded
  size</dfn> (default 0)
  <dd>A number.
+<dt><dfn export for="response body info" id="response-body-info-content-type">content
+ type</dfn> (default empty string)
+ <dd>A string.
```

Comment:
I don't think extracting a MIME type returns a string.

Reply:
I think it would be cleanest btw if we store null or a MIME type here and let callers serialize as needed.

Reply:
Updated to hold a MIME type.